showcase/package.json

```diff
@@ -18,7 +18,7 @@
     "langium-cli": "~1.2.0"
   },
   "dependencies": {
-    "sprotty": "^0.13.0",
+    "sprotty": "0.14.0-next.e058bb2.23",
     "sprotty-elk": "^0.13.1",
```

Comment:
Please use the same version for `sprotty-elk` to avoid issues with dependency resolution.